migrations/v2.js

```diff
@@ -0,0 +1,225 @@
+    assessments.forEach(assessment => {
+      if (assessment._isPercentageBased !== true) assessment._isPercentageBased = true;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._isPercentageBased attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._isPercentageBased === true
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _isPercentageBased has been modified to true');
+    return true;
+  });
+
+  /**
+   * * Modify existing attribute to new value.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._canShowFeedback', async () => {
+    assessments.forEach(assessment => {
+      if (assessment._canShowFeedback !== false) assessment._canShowFeedback = false;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._canShowFeedback attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._canShowFeedback === false
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _canShowFeedback has been modified to false');
+    return true;
+  });
+
+  /**
+   * * Modify existing attribute to new value.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._canShowMarking', async () => {
+    assessments.forEach(assessment => {
+      if (assessment._canShowMarking !== false) assessment._canShowMarking = false;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._canShowMarking attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._canShowMarking === false
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _canShowMarking has been modified to false');
+    return true;
+  });
+
+  /**
+   * * Modify existing attribute to new value.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._canShowModelAnswer', async () => {
+    assessments.forEach(assessment => {
+      if (assessment._canShowModelAnswer !== false) assessment._canShowModelAnswer = false;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._canShowModelAnswer attribute', async () => {
+    const isValid = assessments.every(assessment =>
+      assessment._canShowModelAnswer === false
+    );
+    if (!isValid) throw new Error('adapt-contrib-assessment - _canShowModelAnswer has been modified to false');
+    return true;
+  });
+
+  /**
+   * * Remove existing attribute.
+   */
+  mutateContent('adapt-contrib-assessment - add assessment._requireAssessmentPassed', async () => {
+    assessments.forEach(assessment => {
+      delete assessment._requireAssessmentPassed;
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-assessment - check assessment._requireAssessmentPassed attribute', async () => {
+    const isValid = assessments.every(assessment =>
```

Comment:
Just to go with convention, perhaps: `const isValid = assessments.every(assessment => !_.has(assessment, '_requireAssessmentPassed'));`

Reply:
Thanks. Not super familiar with lodash helpers. But will apply this to any delete modification.